test/fake-timers-test.js

```diff
@@ -4655,6 +4655,23 @@ describe("issue #315 - praseInt if delay is not a number", function () {
     });
 });
 
+describe("#368 - timeout.refresh setTimeout arguments", function () {
+  it("should forward  argumnts passed to setTimeout", function () {
+    var clock = FakeTimers.install();
+    var stub = sinon.stub();
+
+    if (typeof setTimeout(NOOP, 0) === "object") {
```

Comment:
Why is this needed? Is this some kind of check for whether the setTimeout object was installed or not? I think this should be removed, as the test would otherwise crash (rightfully so). If the timer is not installed, then we have other issues, TBH :)
```suggestion
```

Suggest removing this, but I might be not understanding something obvious here (,but then add a comment at least :) )

Reply:
I agree with you. I just saw this in other tests and thought it was needed. Without it, the test crashes. But only on the cli, if I run it with `--inspect-brk` and open the dev tools it doesn't. I don't understand this either.

Reply:
> Why is this needed? Is this some kind of check for whether the setTimeout object was installed or not?

This is a check if you have Node.js timers (objects) or web timers (numbers)

Reply:
Because `.refresh` does not exist on web timers

Reply:
It would be more obvious if the test was conditionally skipped, like be do here:

https://github.com/sinonjs/fake-timers/blob/54852f6d270295f164e7c1d8c217955523e22cfb/test/fake-timers-test.js#L450-L454

Reply:
Yeah, we usually do feature detection and skip tests (or a whole suite) by using Mocha's `Context#skip()` feature. That makes each test more succinct and easier to read.

Reply:
> Because `.refresh` does not exist on web timers

Thanks for the clarification

Reply:
> It would be more obvious if the test was conditionally skipped, like we do here:
> 
> https://github.com/sinonjs/fake-timers/blob/54852f6d270295f164e7c1d8c217955523e22cfb/test/fake-timers-test.js#L450-L454

I'll change it. Thank you for the suggestion.